### PR TITLE
Revert "with Calendar and List"

### PR DESCRIPTION
**Pull request by minadedijer**
Reverts peter-phillips/Learn-By-Planning#31

### CODE REVIEW

File: frontend/src/Calendar.jsx [hunk 1]
```diff
@@ -1,42 +1,10 @@
...
-        events={[
-          { title: 'CSC 307: Midterm 2', date: '2021-05-26' },
-          { title: 'CPE 316: Midterm 2', date: '2021-05-17' },
-        ]}
-        eventColor="#378006"
-        */
-      />
-    );
-  }
+// import styles from './Calendar.module.css';
```
Comment (kmgarcia23) [inline frontend/src/Calendar.jsx:2]: Why is the calendar component not being used?

**Review by kmgarcia23 — COMMENTED** [review]
LGTM!

**Review by peter-phillips — COMMENTED** [review]
Other than outstanding comments looks good to me

File: frontend/src/Calendar.jsx (hunk 1 above)
Comment (jhbanh) [reply frontend/src/Calendar.jsx:2]: ^^^

**Review by jhbanh — COMMENTED** [review]
Why did you take out the Full calendar component?

**Pull request closed without merging**